tamr_client/instance.py

```diff
@@ -10,3 +10,16 @@ def origin(instance: Instance) -> str:
         return f"{instance.protocol}://{instance.host}"
     else:
         return f"{instance.protocol}://{instance.host}:{instance.port}"
+
+
+def version(session: Session, instance: Instance) -> str:
+    """Return the Tamr version for an instance.
+
+    Args:
+        session: Tamr Session
+        instance: Tamr instance
+
+    Returns: Version
+
+    """
+    return session.get(f"{origin(instance)}/api/versioned/service/version").json()["version"]
```

Comment:
Might be nice to add a comment for future contributors:

```suggestion
    # Version endpoints are not themselves versioned by design, but they are stable so they are ok to use here.
    return session.get(f"{origin(instance)}/api/versioned/service/version").json()["version"]
```